packages/firestore/package.json

```diff
@@ -57,7 +60,9 @@
   },
   "peerDependencies": {
     "@firebase/app": "0.x",
-    "@firebase/app-types": "0.x"
+    "@firebase/app-exp": "0.x",
+    "@firebase/app-types": "0.x",
+    "@firebase/app-types-ext": "0.x"
```

Comment:
Should not add -exp packages in dependencies or peerDependencies, because they don't really exist on npm. For now, can you just suppress the eslint errors?

Reply:
Done